core/UserManager/UserHandler.py

```diff
@@ -44,3 +44,28 @@ def handler_users_list(request):
         print(user_list)
         ser = UserSerializer(user_list, many=True)
         return Response(data=ser.data, status=status.HTTP_200_OK)
+
+    @staticmethod
+    def handler_user_registration(request_data):
```

Comment:
Hey, the code looks good, but since we are using RestAPI we must use serializers when hadling Django Models.